index.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="Description" content="Multilingual object detection, &lt;6MB. Works offline.">
+
+    <title>What's That?</title>
+
+    <link rel="stylesheet" type="text/css" href="/assets/styles.css" media="all">
+    <link rel="apple-touch-icon" sizes="180x180" href="/assets/apple-touch-icon.png">
+    <link rel="icon" type="image/png" sizes="32x32" href="/assets/favicon-32x32.png">
+    <link rel="icon" type="image/png" sizes="16x16" href="/assets/favicon-16x16.png">
+    <link rel="manifest" href="/assets/site.webmanifest">
+
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="msapplication-TileColor" content="#54457f">
+    <meta name="theme-color" content="#54457f">
+
+    <script src="/scripts/register-service-worker.js" async></script>
+    <link rel="preload" href="/scripts/web-worker.js" as="script">
```

Comment:
I'm not sure if this works, lighthouse flagged it on a localhost test and idk what's up